.editorconfig

```diff
@@ -1,5 +1,2 @@
-[*.gradle]
-indent_style = tab
-
-[*.java]
+[*.{gradle,mapping}]
```

Comment:
Why not ` indent_size = 4` as well like in your 1.14.4 pr?

Reply:
Its the default, it was late at night for me when I wrote the ones for 1.14 😓